### ISSUE TITLE
SEO Title Source set to Custom Text won't save

### ISSUE THREAD

**Issue by sjelfull** [nystudio107/craft-seomatic#573]
When inputting custom text in the SEO Title Source on a entry via the SEO field, it won't save it. If I type `Hello`, the saved value will be `` (empty string).

This is the value from the content table:

```json
{
  "bundleVersion": "1.0.22",
  "sourceBundleType": "field",
  "sourceId": null,
  "sourceName": null,
  "sourceHandle": null,
  "sourceType": "field",
  "sourceTemplate": "",
  "sourceSiteId": null,
  "sourceAltSiteSettings": [],
  "sourceDateUpdated": "2020-03-05T10:44:07+01:00",
  "metaGlobalVars": {
    "language": null,
    "mainEntityOfPage": "",
    "seoTitle": "",
    "siteNamePosition": "",
    "seoDescription": "",
    "seoKeywords": "",
    "seoImage": "{seomatic.helper.socialTransform(object.entry.newsCover[0], \"base\", 0, \"fit\")}",
    "seoImageWidth": "{seomatic.helper.socialTransformWidth(object.entry.newsCover[0], \"base\", 0, \"fit\")}",
    "seoImageHeight": "{seomatic.helper.socialTransformHeight(object.entry.newsCover[0], \"base\", 0, \"fit\")}",
    "seoImageDescription": "",
    "canonicalUrl": "",
    "robots": "",
    "ogType": "",
    "ogTitle": "",
    "ogSiteNamePosition": "",
    "ogDescription": "",
    "ogImage": "",
    "ogImageWidth": "",
    "ogImageHeight": "",
    "ogImageDescription": "",
    "twitterCard": "",
    "twitterCreator": "",
    "twitterTitle": "",
    "twitterSiteNamePosition": "",
    "twitterDescription": "",
    "twitterImage": "",
    "twitterImageWidth": "",
    "twitterImageHeight": "",
    "twitterImageDescription": ""
  },
  "metaSiteVars": {
    "siteName": "United States",
    "identity": null,
    "creator": null,
    "twitterHandle": "",
    "facebookProfileId": "",
    "facebookAppId": "",
    "googleSiteVerification": "",
    "bingSiteVerification": "",
    "pinterestSiteVerification": "",
    "sameAsLinks": [],
    "siteLinksSearchTarget": "",
    "siteLinksQueryInput": "",
    "additionalSitemapUrls": [],
    "additionalSitemapUrlsDateUpdated": null,
    "additionalSitemaps": []
  },
  "metaSitemapVars": {
    "sitemapUrls": true,
    "sitemapAssets": true,
    "sitemapFiles": true,
    "sitemapAltLinks": true,
    "sitemapChangeFreq": "weekly",
    "sitemapPriority": 0.5,
    "sitemapLimit": null,
    "structureDepth": null,
    "sitemapImageFieldMap": [],
    "sitemapVideoFieldMap": []
  },
  "metaContainers": {
    "MetaTagContainergeneral": {
      "data": [],
      "name": "General",
      "description": "General Meta Tags",
      "class": "nystudio107\\seomatic\\models\\MetaTagContainer",
      "handle": "general",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaTagContaineropengraph": {
      "data": [],
      "name": "Facebook",
      "description": "Facebook OpenGraph Meta Tags",
      "class": "nystudio107\\seomatic\\models\\MetaTagContainer",
      "handle": "opengraph",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaTagContainertwitter": {
      "data": [],
      "name": "Twitter",
      "description": "Twitter Card Meta Tags",
      "class": "nystudio107\\seomatic\\models\\MetaTagContainer",
      "handle": "twitter",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaTagContainermiscellaneous": {
      "data": [],
      "name": "Miscellaneous",
      "description": "Miscellaneous Meta Tags",
      "class": "nystudio107\\seomatic\\models\\MetaTagContainer",
      "handle": "miscellaneous",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaLinkContainergeneral": {
      "data": [],
      "name": "General",
      "description": "Link Tags",
      "class": "nystudio107\\seomatic\\models\\MetaLinkContainer",
      "handle": "general",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaScriptContainergeneral": {
      "data": [],
      "position": 1,
      "name": "General",
      "description": "Script Tags",
      "class": "nystudio107\\seomatic\\models\\MetaScriptContainer",
      "handle": "general",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaJsonLdContainergeneral": {
      "data": [],
      "name": "General",
      "description": "JsonLd Tags",
      "class": "nystudio107\\seomatic\\models\\MetaJsonLdContainer",
      "handle": "general",
      "include": true,
      "dependencies": [],
      "clearCache": false
    },
    "MetaTitleContainergeneral": {
      "data": [],
      "name": "General",
      "description": "Meta Title Tag",
      "class": "nystudio107\\seomatic\\models\\MetaTitleContainer",
      "handle": "general",
      "include": true,
      "dependencies": [],
      "clearCache": false
    }
  },
  "redirectsContainer": [],
  "frontendTemplatesContainer": {
    "data": [],
    "name": null,
    "description": null,
    "class": "nystudio107\\seomatic\\models\\FrontendTemplateContainer",
    "handle": null,
    "include": true,
    "dependencies": null,
    "clearCache": false
  },
  "metaBundleSettings": {
    "siteType": "",
    "siteSubType": "",
    "siteSpecificType": "",
    "seoTitleSource": "fromCustom",
    "seoTitleField": "title",
    "siteNamePositionSource": "sameAsGlobal",
    "seoDescriptionSource": "fromField",
    "seoDescriptionField": "newsBlocks",
    "seoKeywordsSource": "fromCustom",
    "seoKeywordsField": "subline",
    "seoImageIds": [
      "8126"
    ],
    "seoImageSource": "fromField",
    "seoImageField": "newsCover",
    "seoImageTransform": "1",
    "seoImageTransformMode": "fit",
    "seoImageDescriptionSource": "fromField",
    "seoImageDescriptionField": "title",
    "twitterCreatorSource": "sameAsSite",
    "twitterCreatorField": "",
    "twitterTitleSource": "fromField",
    "twitterTitleField": "title",
    "twitterSiteNamePositionSource": "sameAsGlobal",
    "twitterDescriptionSource": "fromField",
    "twitterDescriptionField": "newsBlocks",
    "twitterImageIds": [],
    "twitterImageSource": "sameAsSeo",
    "twitterImageField": "",
    "twitterImageTransform": true,
    "twitterImageTransformMode": "crop",
    "twitterImageDescriptionSource": "sameAsSeo",
    "twitterImageDescriptionField": "",
    "ogTitleSource": "fromField",
    "ogTitleField": "title",
    "ogSiteNamePositionSource": "sameAsGlobal",
    "ogDescriptionSource": "fromField",
    "ogDescriptionField": "newsBlocks",
    "ogImageIds": [],
    "ogImageSource": "sameAsSeo",
    "ogImageField": "",
    "ogImageTransform": true,
    "ogImageTransformMode": "crop",
    "ogImageDescriptionSource": "sameAsSeo",
    "ogImageDescriptionField": ""
  }
}
```

**Comment by sjelfull** [comment]
Just verified that this is fixed by #570

**Comment by khalwat** [comment]
Fixed then!
